docs/pages/management/operations/self-hosted-automatic-agent-updates.mdx

```diff
@@ -27,12 +27,9 @@ already done, or you are a Teleport Cloud user, you can directly
 Systemd agents enrolled into automatic updates can only install versions
 present in their package repositories. As Teleport 14 won't be published to
 `stable/v13`, those agents will require manual intervention to be updated to
-the next major version (adding a new apt/yum repo for `stable/v14`).
-
-This limitation will be fixed before Teleport 14. We'll provide a rolling
-update channel for agents to be able to automatically update from one major
-to another. When this will happen, you will have to update the repos of the
-existing agents to point to the new repo.
+the next major version (adding a new APT/YUM/zypper repo for `stable/v14`).
+Alternatively, the `stable/rolling` channel can be used which contains
```

Comment:
The way we mention `stable/rolling` channel feels very "and by the way you can use this channel" which I think might be easy to miss unless you actually read the entire guide carefully.

I wonder if it would be useful to have a table somewhere on Installation page that describes what channels we have and what gets published to them. Then users can decide which one they want to use. @fheinecke @ptgott What do you guys think?

Something like:

```
Channel name    | Description
---------------------------
stable/<major>  | Receives releases for a particular major release line
stable/cloud    | Rolling channel that receives releases compatible with current Cloud version
stable/rolling  | Rolling channel that receives all published Teleport releases
```

Reply:
That sounds good to me!